extensions/amp-date-picker/0.1/amp-date-picker.css

```diff
@@ -14,8 +14,234 @@
+  border: 1px solid transparent;
+}
+
+.CalendarDay__hovered_span,
+.CalendarDay__hovered_span:hover,
+.CalendarDay__selected_span,
+.CalendarDay__selected_span:active,
+.CalendarDay__selected_span:hover {
+  background-color: #9bbbdd;
+  border: 1px solid #9bbbdd;
+  color: #000;
+}
+
+.CalendarDay__selected,
+.CalendarDay__selected_start,
+.CalendarDay__selected:active,
+.CalendarDay__selected:hover {
+  background-color: #0654ab;
+  border: 1px solid #0654ab;
+  color: #fff;
+}
+
+.DayPickerNavigation_leftButton__horizontal,
+.DayPickerNavigation_leftButton__horizontal:active {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="25" xmlns="http://www.w3.org/2000/svg"><path fill="#fff" d="M15.7 22l-9.5-9.5L15.7 3l-3-3L.4 12.6 12.8 25"/></svg>');
+}
+
+.DayPickerNavigation_rightButton__horizontal,
+.DayPickerNavigation_rightButton__horizontal:active {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="25" xmlns="http://www.w3.org/2000/svg"><path fill="#fff" d="M.3 3l9.5 9.5L.3 22l3 3 12.4-12.5L3.2 0"/></svg>')!important;
+}
+
+.DayPickerNavigation_button__horizontal,
+.DayPickerNavigation_button__horizontal:hover,
+.DayPickerNavigation_button__horizontal:active {
+  border: 0;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  background-position: 55% 50%;
+  height: 30px;
+  width: 26px;
+  background-size: 13px 20px;
+}
+
+.DayPickerNavigation_svg__horizontal {
+  display: none;
+}
+
+.DateInput_fangShape {
+  fill: #000;
+}
+
+.DayPickerKeyboardShortcuts_show,
+.DayPickerKeyboardShortcuts_show:hover {
+  border: 2px solid #0654ab;
+  border-radius: 50%;
+  color: #0654ab;
+  width: 22px;
+  height: 22px;
+  margin: 10px;
+}
+
+.DayPickerKeyboardShortcuts_show:hover {
+  border: 2px solid #9bbbdd;
+}
+
+.DayPickerKeyboardShortcuts_show:hover .DayPickerKeyboardShortcuts_showSpan {
+  color: #9bbbdd;
+}
+
+.DayPickerKeyboardShortcuts_showSpan {
+  position: static;
+  color: #0654ab;
+  font-weight: bold;
+}
+
+.CalendarDay__highlighted_calendar,
+.CalendarDay__highlighted_calendar:hover {
+  background-image: radial-gradient(circle at 50% 82%, #0654ab 0px, #0654ab 2px, transparent 3px, transparent 100%);
+}
+
+.CalendarDay__selected.CalendarDay__highlighted_calendar {
+  background-image: radial-gradient(circle at 50% 82%, #fff 0px, #fff 2px, transparent 3px, transparent 100%);
+}
+
+.CalendarDay__blocked_out_of_range.CalendarDay__highlighted_calendar::after {
+  background-image: radial-gradient(circle at 50% 82%, rgba(0, 0, 0, 0.1) 0px, rgba(0, 0, 0, 0.1) 2px, transparent 3px, transparent 100%);
+}
+
+.i-amphtml-amp-date-picker-info {
+  display: block;
+  padding: 10px;
+}
+
+/**
+ * TODO(cvializ): scope this to the date picker unless we want to enable AMP
+ * tooltips more widely ; ) This is ok now because amp-date-picker is still
+ * experimental
+ * Adapted from: https://freqdec.github.io/Pure-CSS-Tooltips/
+ */
+
+[data-tooltip] {
```

Comment:
we should scope it. Even if we make an amp version, sill need to prefix with `amp`. `[data-tooltip]` too generic and can collide with user defined css easily.

Reply:
Agreed. Will do